SingularityBase/src/main/java/com/hubspot/singularity/SingularityRequestHistory.java

```diff
@@ -31,7 +31,9 @@
     DEPLOYED_TO_UNPAUSE,
     BOUNCED,
     SCALED,
-    SCALE_REVERTED
+    SCALE_REVERTED,
+    PRIORITIZED,
+    PRIORITY_REVERTED
```

Comment:
are we able to maybe fold these into UPDATED instead somehow? these fields come through on things that a lot of other systems pull in and the roll out can be a pain with the deployer/etc having to know about all the new fields first. If the purpose is just to indicate the action more easily, we can always have the messge text reflect that, which will also show in the UI